src/features/todo/Todo.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { createTask, isCompleted, editTask, deleteTask } from './todoSlice'
+import { useState } from 'react'
+
+export default function Todo() {
+    const [discription, setDiscribtion] = useState('')
+    const [editText, setEditText] = useState('')
+    const [isTaskToEdit, setIsTaskToEdit] = useState(-1)
+    const dispatch = useDispatch()
+    const todo = useSelector(state => state.todo)
+
+    console.log(todo)
+    
+    const handleSubmit = (e)=> {
+        e.preventDefault();
+        dispatch(createTask(discription))
+        setDiscribtion('')
+    }
+    const handleDelete =(id) => () =>{
```

Comment:
We are you using higher order functions here? Why return an extra function?

The function definition can simply be:

```js
const handleDelete =(id) => {
  dispatch(deleteTask(id));
}
```

and it's usage in line 56 could be like this:

```jsx
<button onClick={() => handleDelete(task.id)}>Delete</button>
```